tao_compiler/mlir/disc/transforms/disc_dot_merge.cc

```diff
@@ -35,6 +35,52 @@
 namespace mlir {
 namespace disc_ral {
 namespace {
+//getMNDim get the m-dim and n-dim of the input dot of (m,k)x(k,n)   
+std::pair<int64_t,int64_t> getMNDim(mhlo::DotGeneralOp op) {
```

Comment:
This function is only used by `DotShareOperandMergeConverter`. Better to make it private.